crates/typos-cli/src/file_type_specifics.rs

```diff
@@ -0,0 +1,66 @@
+//! This module specifies [`EngineConfig`] defaults for the file types defined in [`default_types`].
+//!
+//! [`EngineConfig`]: crate::config::EngineConfig
+//! [`default_types`]: crate::default_types
+
+/// Set `check_file` to `false` for these types.
+pub const NO_CHECK_TYPES: &[&str] = &["cert", "lock"];
+
+pub const TYPE_SPECIFIC_DICTS: &[(&str, StaticDictConfig)] = &[
+    (
+        "py",
+        StaticDictConfig {
+            ignore_idents: &["NDArray"],
+            ignore_words: &[],
+        },
+    ),
+    (
+        "rust",
+        StaticDictConfig {
+            ignore_idents: &["flate2"],
+            ignore_words: &["ser"],
+        },
+    ),
+    (
+        "vim",
+        StaticDictConfig {
+            ignore_idents: &["windo"],
+            ignore_words: &[],
+        },
+    ),
+    (
+        "vimscript",
+        StaticDictConfig {
+            ignore_idents: &["windo"],
+            ignore_words: &[],
+        },
+    ),
+];
+
+pub struct StaticDictConfig {
+    pub ignore_idents: &'static [&'static str],
+    pub ignore_words: &'static [&'static str],
+}
+
+#[cfg(test)]
+#[test]
```

Comment:
nit: technically, this is redundant

Reply:
Oh ... I didn't know that. Yeah that makes sense. I removed the `#[cfg(test)]`.